docs/man/git-lfs-config.5.ronn

```diff
@@ -111,8 +111,9 @@ be scoped inside the configuration for a remote.
 * `lfs.largefilewarning`
 
   Warn when a file is 4 GiB or larger. Such files will be corrupted when using
-  Windows (unless smudging is disabled) due to a limitation in Git.  Default:
-  true.
+  Windows (unless smudging is disabled) with a Git for Windows version less than
+  2.34.0 due to a limitation in Git.  Default: true if the version is less than
+  2.34.0, false otherwise.
 
```

Comment:
I would also suggest changing the line in "Known Bugs" [section](https://github.com/git-lfs/git-lfs/blob/b197e981b5da134bdc5a94db3b23d5179f9ad6ed/docs/man/git-lfs-smudge.1.ronn#known-bugs) of `docs/man/git-lfs-smudge.1.ronn` which reads:
> `On Windows, Git does not handle files in the working tree larger than 4 gigabytes.`
>
> `For more information, see: https://github.com/git-lfs/git-lfs/issues/2434.`

Reply:
Updated.  I've removed the link to the bug and stated that newer versions are unaffected.